src/components/ConfirmDialog.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button, Dialog, DialogActions, DialogContent } from '@mui/material';
+
+export default function ConfirmDialog({ open, prompt, onConfirm, onClose }: { open: boolean; prompt: string; onConfirm: () => void; onClose: () => void }) {
+  const handleConfirm = () => {
+    onConfirm();
+    onClose();
+  };
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <DialogContent>{prompt}</DialogContent>
+      <DialogActions>
+        <Button onClick={onClose}>Cancel</Button>
+        <Button variant="contained" onClick={handleConfirm}>
+          confirm
```

Comment:
casing. Is "OK" more common?

Reply:
Button converts it to all uppercase. I am aligned on "OK", will swap that out.